src/Stratis.Bitcoin.Features.RPC/RpcSettings.cs

```diff
@@ -32,74 +32,118 @@ public class RpcSettings
         public int RPCPort { get; set; }
 
         /// <summary>Default bindings from config.</summary>
-        public List<IPEndPoint> DefaultBindings { get; set; }
+        public List<IPEndPoint> DefaultBindings => DefaultBinding(this.AllowIp.Any(), this.RPCPort);
```

Comment:
Maybe its better to leave the get/set here and set it once as every time we call `DefaultBindings` it will execute the method.

Reply:
Currently it's only used once by the private method `LoadSettingsFromConfig`. If the user needs to use it for settings setup then we need it to recalculate as the user my have modified the `RpcPort` value.